### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.10)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29372]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Ryan Thurlwell (OCTO Design Lead)

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
"Remove" is a text link

### Link, screenshot or steps to recreate
https://staging.va.gov/gi-bill-comparison-tool-sandbox/compare/?facilities=14952435%2C10040835%2C21853035

### VA.gov Experience Standard
[Cateogy Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
This should be a secondary button

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by amyknox** [comment]
Changing "Remove" text link to secondary button

**Comment by watersemma** [comment]
Updates have been made and can be seen here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/education-careers/school-comparison-tool/redesign/design/sketch-files/gibct-redesign-comparison-postmvp.sketch

**Comment by af-manish** [comment]
@watersemma Can you look at @rtwell 's comment and update designs appropriately?

**Comment by ERRonat0609** [comment]
@af-manish Emma Waters will not be updating design documents - she was on the BAH team that we took over from, so she is no longer on this project.

**Comment by ERRonat0609** [comment]
@af-manish @mrblanco1  - After looking at this again, if we know that this update is correct except the buttons need to be inside the gray boxes and left-aligned with the school name text, do we really need a new mockup? 
![image](https://user-images.githubusercontent.com/85580424/144675421-082e151e-c6f4-4694-9bee-1658974793e2.png)

**Comment by hallihocine** [comment]
- Test Run in Test Rail:
https://dsvavsp.testrail.io/index.php?/runs/view/2819&group_by=cases:section_id&group_order=asc&group_id=3319
- Test Script/Steps: 
[Design components or patterns don't align with Design System guidelines. (04.07.10).xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/7847832/Design.components.or.patterns.don.t.align.with.Design.System.guidelines.04.07.10.xlsx)
